_data/sites.json

```diff
@@ -5191,7 +5191,16 @@
             "support.native-instruments.com"
         ]
     },
-
+    {
+        "name": "NCBI",
+        "url": "https://www.ncbi.nlm.nih.gov/books/NBK3842/",
+        "difficulty": "impossible",
```

Comment:
I'd say it is hard. Though it takes time it is still done after a while

Reply:
okay, I'm gonna change that line